### DISCUSSION TITLE
open source my repository

### DISCUSSION THREAD

**Discussion by d-coder111** [community/community#18928] [Other Feature Feedback, Questions, & Ideas] [answered]
How to open source my repo so that people can contribute to it and connect with me ?

**Comment by Tran-Quyen** [comment]
You can make issues to everyone can contribute to your repo

**Reply by d-coder111** [reply]
I am kinda new to the community so how will people know about the it.

**Reply by Tran-Quyen** [reply]
@d-coder111 You can simply make discussion like this post.

**Reply by d-coder111** [reply]
So mainly if I want to open source my repo to connect with more people,then will I have to post it under discussion?

**Reply by Tran-Quyen** [reply]
> So mainly if I want to open source my repo to connect with more people,then will I have to post it under discussion?

Yes. You need member join open source and have detail document about your app you want to develop.

**Reply by Tran-Quyen** [reply]
You can start with this course: https://lab.github.com/githubtraining/create-an-open-source-program

**Reply by d-coder111** [reply]
ok thnx😊

**Comment by rambleraptor** [comment]
I'd recommend looking through the [GitHub tutorial](https://docs.github.com/en/get-started/quickstart/hello-world)

**Comment by melvincwng** [comment, marked as answer]
Hi @d-coder111, first of all, let me welcome you to GitHub Community.

Just some suggestions on how to increase your outreach of your open source projects.

1. Blog on medium.com about your open source project
2. Share it with friends/family on various social media platforms
3. Attend and provide talks about your project at physical events 
4. Post about your open source projects and its goals on various discussion platforms such as this
5. Create a website about your project which helps to solve a real world problem.

Having said all that, it wouldn't be easy trying to standout against the many established open source projects on GitHub.

I sincerely wish you all the best in your journey of creating your own open source project. I am also working on developing my own open source project for the community as well :).

Good luck and keep on coding 💪

**Reply by queenofcorgis** [reply]
Appreciate the detailed and supportive answer @melvincwng :octocat:

**Reply by melvincwng** [reply]
Hi @liligalante, just trying to help out fellow members of the GitHub Community 😊

**Reply by melvincwng** [reply]
Hey @d-coder111, thanks for connecting on GitHub 😄 

If my reply is helpful, please kindly 'Mark as answer' so others know this issue has been addressed.

Hope we can collaborate in the future on some open source projects.

Take care, and have a wonderful day ahead 👍🏻

**Reply by d-coder111** [reply]
sure sure😊👏

**Reply by trnvnphuong1** [reply]
Phuong tran & *** redacted due to PII ***

**Reply by YAB99GitHub** [reply]
Please, I am new to this great GitHub. I am to write on an ISSUE section, but I don't know how to start with the write-up. Any transparent clarification about my issue? Thank you all participants.

**Comment by trnvnphuong1** [comment]
Tranphuong.com.vn 
[Backup_codes.txt](https://github.com/community/community/files/9445901/Backup_codes.txt)
